src/lib/components/Header.svelte

```diff
@@ -0,0 +1,95 @@
+<script type="ts">
+  import { NAV_ELEMENTS } from "$lib/config/homePageData";
+
+  import { fly } from "svelte/transition";
+
+  import AnimatedHumburger from "./AnimatedHumburger.svelte";
+
+  export let open = false;
+  export let onClick = (): void => {
+    open = !open;
+
+    if (open) {
+      document.body.style.overflow = "hidden";
+    } else {
+      document.body.style.overflow = "scroll";
+    }
+  };
+</script>
+
+<header>
+  <section class="container">
+    <div class="desktop-menu surface-tint header">
+      <a href="/" class="on-secondary-text">EME STUDIO</a>
+      <nav>
+        <ul>
+          {#each NAV_ELEMENTS as navElements}
```

Comment:
Same comment as the Footer {#each} block here..
`navElements` it's not plural..